loverajoel/magicplaylist
# New design

What does this Pull Request Do?
-------------------------------
Main changes:
* New design in general
* Remove `svg` images and use Font Awesome icons instead
* Use `sass` instead of vanilla `css` for the styles
* Use more "semantic" html tags where needed. ex:  `<p>` instead of `<span>` for paragraphs.

:warning: WIP :warning: 

How should this be manually tested?
-----------------------------------
Checkout branch > `npm install` > `npm run dev` and check that everything works but with the new design
